src/components/cdPipeline/CDPipeline.tsx

```diff
@@ -575,11 +599,10 @@ export default class CDPipeline extends Component<CDPipelineProps, CDPipelineSta
             this.validationRules.name(this.state.pipelineConfig.name).isValid &&
```

Comment:
Move this validation before any computation

Reply:
As we're already checking for pipeline name & namespace above it makes sense to merge both